ocs_ci/ocs/tests/test_fiojob.py

```diff
@@ -0,0 +1,472 @@
+                "1" : 100.000000,
+                "2" : 0.000000,
+                "4" : 0.000000,
+                "8" : 0.000000,
+                "16" : 0.000000,
+                "32" : 0.000000,
+                ">=64" : 0.000000
+              },
+              "latency_ns" : {
+                "2" : 0.000000,
+                "4" : 0.000000,
+                "10" : 0.000000,
+                "20" : 0.000000,
+                "50" : 0.000000,
+                "100" : 0.000000,
+                "250" : 0.000000,
+                "500" : 0.000000,
+                "750" : 92.436298,
+                "1000" : 6.599584
+              },
+              "latency_us" : {
+                "2" : 0.744950,
+                "4" : 0.044621,
+                "10" : 0.117499,
+                "20" : 0.040293,
+                "50" : 0.015126,
+                "100" : 0.010000,
+                "250" : 0.010000,
+                "500" : 0.010000,
+                "750" : 0.010000,
+                "1000" : 0.010000
+              },
+              "latency_ms" : {
+                "2" : 0.010000,
+                "4" : 0.000000,
+                "10" : 0.000000,
+                "20" : 0.000000,
+                "50" : 0.000000,
+                "100" : 0.000000,
+                "250" : 0.000000,
+                "500" : 0.000000,
+                "750" : 0.000000,
+                "1000" : 0.000000,
+                "2000" : 0.000000,
+                ">=2000" : 0.000000
+              },
+              "latency_depth" : 1,
+              "latency_target" : 0,
+              "latency_percentile" : 100.000000,
+              "latency_window" : 0
+            }
+          ],
+          "disk_util" : [
+            {
+              "name" : "rbd1",
+              "read_ios" : 42,
+              "write_ios" : 18382,
+              "read_merges" : 0,
+              "write_merges" : 18322,
+              "read_ticks" : 33031,
+              "write_ticks" : 50421049,
+              "in_queue" : 50444926,
+              "util" : 7.917599
+            }
+          ]
+        }
+        """)
+    return fio_out
+
+
+@pytest.fixture
+def fio_json_output_with_error(fio_json_output):
+    """
+    Example of fio --output-format=json output, with io_u error. Based on
+    actual test run.
+    """
+    err_line = (
+        "fio: io_u error on file /mnt/target/simple-write.0.0: "
+        "No space left on device: write offset=90280222720, buflen=4096")
+    return err_line + "\n" + fio_json_output
+
+
+@pytest.fixture
+def fio_json_output_broken():
+    """
+    Example of broken, unparseable json output.
+    """
+    fio_out = textwrap.dedent("""
+        {
+          "fio version" : "fio-3.7",
+          "timestamp" : 1584531581,
+          "timestamp_ms" : 1584531581691,
+          "time" : "Wed Mar 18 11:39:41 2020",
+          "jobs" : [
+        """)
+    return fio_out
+
+
+def test_fio_to_dict_empty():
+    assert fiojob.fio_to_dict("") is None
```

Comment:
add assertion messages, here and in below assert usages

Reply:
This is unittest of ocs-ci module code, and it covers very simple edge case, documenting the behavior what will happen when one calls the function with an empty string. I don't think there is anything to be added on top of that. Unit tests like these are useful to make sure edge cases doesn't fail in unexpected ways, and that this property will hold when the code is edited later.